### PR TITLE
Add before/after content element

### PR DESCRIPTION
**Pull request by aviav**
REDMINE-17372

### CODE REVIEW

File: entry_types/scrolled/package/src/contentElements/inlineBeforeAfter/BeforeAfter.js [hunk 1]
```diff
@@ -1,99 +1,107 @@
...
+    <div ref={beforeAfterRef}
+      className={cx({[styles.selected]: isSelected, [styles.wiggle]: wiggle},
+      styles.container)}>
+      <InitialSliderPositionIndicator parentSelected={isSelected}
+        position={initial_slider_position}/>
+      {/* onSliderPositionChange: Prevent wiggle if user uses slider */}
+      <ReactCompareImage leftImage={beforeImageUrl} rightImage={afterImageUrl}
+                         leftImageLabel={before_label} rightImageLabel={after_label}
+                         leftImageAlt={beforeImageAlt} rightImageAlt={afterImageAlt}
+                         sliderPositionPercentage={initial_slider_position/100}
```
Comment (DanielKuhn) [inline entry_types/scrolled/package/src/contentElements/inlineBeforeAfter/BeforeAfter.js]: Since you're already calculating it up top, you could use `initialSliderPos` here.

File: entry_types/scrolled/package/src/frontend/index.js [hunk 2]
```diff
@@ -25,6 +27,7 @@ export * from './i18n';
 export * from './SectionThumbnail';
 export {default as Entry} from './Entry';
 export {EntryStateProvider} from '../entryState/EntryStateProvider'
+export {useEditorSelection} from './EditorState';
```
Comment (DanielKuhn) [inline entry_types/scrolled/package/src/frontend/index.js]: Why is this needed here? Unused?

File: entry_types/scrolled/package/src/frontend/index.js [hunk 3]
```diff
@@ -14,6 +14,8 @@ const editMode = window.location.pathname.indexOf('/editor/entries') === 0;
 export {api as frontend} from './api';
 
 export * from './Image';
+export * from './useImageUrl';
+export * from './useImageAlt';
```
Comment (DanielKuhn) [inline entry_types/scrolled/package/src/frontend/index.js]: These two need are only needed until `useFile` is fixed for usage in content elements. But for now they can stay here.

File: entry_types/scrolled/package/src/frontend/index.js (hunk 2 above)
Comment (aviav) [reply entry_types/scrolled/package/src/frontend/index.js]: We need to export this here to be able to use it in BeforeAfter.js
